cypress-shared/support/shopper-location/settings.js

```diff
@@ -0,0 +1,46 @@
+import { FAIL_ON_STATUS_CODE } from '../common/constants'
+import { updateRetry } from '../common/support'
+
+const version = '*.x'
+const app = 'vtex.shopper-location'
+
+export function updateSettings(
+  country,
+  url,
+  { automaticRedirect = false } = {}
+) {
+  it(
+    `Configuring automatic redirect with country ${country} in ${app}`,
```

Comment:
If automaticeRedirect is true then 

if automaticRedirect then title should be 
Configuring automatic redirect with country ${country} in ${app}
else
Configuring redirect with country ${country} in ${app}